Kitodo/src/main/webapp/WEB-INF/templates/includes/processes/processList.xhtml

```diff
@@ -351,9 +351,16 @@
                             action="#{ProcessForm.showStateOfVolume}"
                             update="processesTabView:processesForm"
                             icon="fa fa-arrow-circle-o-up"/>
+                <p:menuitem id="durationOfTasks"
+                            value="#{msgs.durationOfTasks}"
+                            action="#{ProcessForm.showDurationOfTasks}"
+                            update="processesTabView:processesForm"
+                            icon="fa fa-arrow-circle-o-up"/>
             </p:menu>
         </div>
-        <p:pieChart rendered="#{ProcessForm.showStatistic}" model="#{ProcessForm.pieModel}" style="width: 100%; height: 500px;"/>
+
+        <p:pieChart rendered="#{ProcessForm.isPieModel}" model="#{ProcessForm.pieModel}" style="width: 100%"/>
+        <p:barChart rendered="#{ProcessForm.isBarModel}" model="#{ProcessForm.stackedBarModel}" style="width: 100%"/>
```

Comment:
```suggestion
        <p:pieChart rendered="#{ProcessForm.pieModel}" model="#{ProcessForm.pieModel}" style="width: 100%"/>
        <p:barChart rendered="#{ProcessForm.barModel}" model="#{ProcessForm.stackedBarModel}" style="width: 100%"/>
```
You must either leave out the `is` or write the methods' parentheses to call a boolean getter.